docusaurus/docs/available-scripts.md

```diff
@@ -6,25 +6,29 @@ sidebar_label: Available Scripts
 
 In the project directory, you can run:
 
-## `npm start`
+## `npm start` or `yarn start`
```

Comment:
It seems like we refer to npm, or Yarn, or both throughout the docs...and then Yarn in the CLI `--help`. I'm thinking that it would be good to look at this throughout the docs and see what makes the most sense.